ClientApp/src/components/reuseables/ProjectEvaluationResults/ProjectEvaluationResults.tsx

```diff
@@ -17,33 +19,86 @@ export const ProjectEvaluationResults: React.FC<ProjectEvaluationResultsProps> =
   subtitle,
   title,
 }: ProjectEvaluationResultsProps) => {
+  const [currentEvalIndex, setCurrentEvalIndex] = useState(-1);
+  const [dropdownTitle, setDropdownTitle] = useState("Select Team Member");
+
+  const onDropdownItemClick = (name: string, id: number): void => {
+    setCurrentEvalIndex(id);
+    setDropdownTitle(name);
+  };
+
   return (
     <>
       <Row>
         <Col>
           <h3>{title || "Evaluation Results"}</h3>
         </Col>
       </Row>
-      {results?.sections.map((section) => (
-        <>
-          <Row>
-            <Col>
-              <h4>
-                {subtitle ||
-                  `Results for ${section.employeeReviewee.firstName} ${section.employeeReviewee.lastName}`}
-              </h4>
-            </Col>
-          </Row>
-          <Row>
-            <Col md={8} xs={12}>
-              <EvaluationResultsTable section={section} />
-            </Col>
-            <Col md={4} xs={12}>
-              <EvaluationNotes notes={section.notes} />
-            </Col>
-          </Row>
-        </>
-      ))}
+      <Row>
+        <Col>
+          <h4>
+            {subtitle || (
+              <EvaluationDropdown
+                onItemClick={onDropdownItemClick}
+                results={results}
+                title={dropdownTitle}
+              />
+            )}
+          </h4>
+        </Col>
+      </Row>
+      {results?.sections.map((section) => {
+        const {
+          employeeReviewee: { employeeId: id },
+          notes,
+        } = section;
+        return (
+          <div key={id} className={currentEvalIndex === id ? "" : "d-none"}>
+            <Row>
+              <Col md={8} xs={12}>
+                <EvaluationResultsTable section={section} />
+              </Col>
+              <Col md={4} xs={12}>
+                <EvaluationNotes notes={notes} />
+              </Col>
+            </Row>
+          </div>
+        );
+      })}
+    </>
+  );
+};
+
+interface EvaluationDropdownProps {
+  onItemClick: (name: string, id: number) => void;
+  results: ProjectEvaluationResultsData | undefined;
+  title: string;
+}
+
+const EvaluationDropdown: React.FC<EvaluationDropdownProps> = ({
+  results,
+  title,
+  onItemClick,
+}: EvaluationDropdownProps) => {
+  return (
+    <>
+      Results for
+      <DropdownButton id="dropdown-basic-button" title={title} variant="info">
+        {results?.sections.map(
+          ({ employeeReviewee: { employeeId: id, firstName, lastName } }) => (
+            <Dropdown.Item
+              key={id}
+              eventKey={`${id}`}
+              onClick={(): void => {
+                // TODO: pretty sure this will change to fullName on the api.
+                onItemClick(`${firstName || ""} ${lastName || ""} `, id ?? -1);
+              }}
+            >
+              {firstName} {lastName}
+            </Dropdown.Item>
+          ),
+        )}
+      </DropdownButton>
```

Comment:
`react-bootstrap` has multiple ways of making a dropdown. To be consistent with other dropdowns in the app, I would suggest 
```suggestion
      <Dropdown>
        <Dropdown.Toggle id="dropdown-basic-button" variant="info">
          {title}
        </Dropdown.Toggle>
        <Dropdown.Menu>
          {results?.sections.map(
            ({ employeeReviewee: { employeeId: id, firstName, lastName } }) => (
              ...
            ),
          )}
        </Dropdown.Menu>
      </Dropdown>
```